.github/workflows/scripts/generate_csr.sh

```diff
@@ -0,0 +1,11 @@
+
+# generate CA cerficate
+openssl genrsa -out fortio.com.key 2048
+openssl req -new -x509 -days 365 -key fortio.com.key -subj "/C=CN/ST=GD/L=SZ/O=fortio.com, Inc./CN=fortio.com Root CA" -out fortio.com.crt
```

Comment:
The address `fortio.com` is come from application we used in our test environment, it may be confusing to other developers. Can you replace all `fortio.com` here with another word, such as the application used in this test?

Reply:
@gyohuangxin so change all the instances that mentions `fortio.com` with `httpbin_app` there are some instances that have ` httpbin.fortio.com.key` so we will change this into `httpbin.httpbin_app.key` right?